src/plugins/auto/utils/log_util.hpp

```diff
@@ -33,6 +33,11 @@
 #define LOG_WARNING(...) HLogPrint(true, false, MultiDevicePlugin::LogLevel::WARN, "WARN", nullptr, __VA_ARGS__)
 #define LOG_ERROR(...) HLogPrint(true, false, MultiDevicePlugin::LogLevel::ERROR, "ERROR", nullptr, __VA_ARGS__)
 
```

Comment:
Add precondition to use following LOG MACROs

Reply:
Usage notes have been added